robottelo/exceptions.py

```diff
@@ -77,6 +77,30 @@ class CapsuleHostError(Exception):
     pass
 
 
+class ContentHostError(Exception):
+    """Indicates error in content configuration."""
+
+    pass
```

Comment:
unless ruff complains about it, you shouldn't need the pass statements in these classes

Reply:
updated the pr